index.pug

```diff
@@ -0,0 +1,96 @@
+html
+  head
+    link(rel="stylesheet" href="/style.css")
+    script(src="../cars.js")
+    body
+     h1.🚦MY CAR COLLECTION API 🏎
+
+     h2. here's a curated list of cars desired by customers /people
+     
+    ul.list 
+      p Get the cars listed 
+
+      li.carList
+        p 
+          strong GET 
+          | /allCars
+        div#getAllCars
+            p Method: GET 
+            p Route: http://localhost:2035/cars/allCars   
+            p Desired outcome : this should bring out all the cars 
+
+      li.listItems 
+            p 
+          strong GET 
+          |/all Cars By Id
+        div#getById
+            p method : GET 
+            p route: http.//localhost:7825/cars/allCars{id}
```

Comment:
Having `allCars` here is weird because it isn't all cars, it's one car. I would expect `/cars/1`